examples/simple-udp/client/main.go

```diff
@@ -44,14 +44,20 @@ func main() {
 	if err != nil {
 		log.Fatalf("Could not connect: %v", err)
 	}
-	defer conn.Close()
+
+	defer func() {
+		err = conn.Close()
+		if err != nil {
+			log.Fatalf("Could not close connection: %v", err)
+		}
+	}()
 
 	go func() {
 		b := make([]byte, bufferSize)
 		for {
-			n, err := conn.Read(b)
-			if err != nil {
-				log.Fatalf("Could not read packet: %v", err)
+			n, errRead := conn.Read(b)
```

Comment:
Catching these kinds of things are why the linters are so helpful.

Reply:
@enocom Does that mean you'll integrate `gometalinter` into the cloudbuild.yaml system? :smile: 

It's already in the build package.

Reply:
Added #163 to track the work.